botorch/utils/gp_sampling.py

```diff
@@ -341,13 +343,15 @@ def get_deterministic_model_list(
     return ModelList(*samples)
 
 
-def get_weights_posterior(X: Tensor, y: Tensor, sigma_sq: float) -> MultivariateNormal:
+def get_weights_posterior(X: Tensor, y: Tensor, sigma_sq: Tensor) -> MultivariateNormal:
     r"""Sample bayesian linear regression weights.
 
     Args:
-        X: a `(batch_shape) x n x num_rff_features`-dim tensor of inputs
-        y: a `(batch_shape) x n`-dim tensor of outputs
-        sigma_sq: the noise variance
+        X: A `(batch_shape) x n x num_rff_features`-dim tensor of inputs.
```

Comment:
```suggestion
        X: A tensor of inputs with shape `(*batch_shape, n, num_rff_features)`.
```
Similar to above, I know we've been writing documentation like this throughout, but I think using Python code to explain this would be clearer